pandas/core/sorting.py

```diff
@@ -527,6 +527,8 @@ def get_indexer_dict(label_list, keys):
     shape = [len(x) for x in keys]
 
     group_index = get_group_index(label_list, shape, sort=True, xnull=True)
+    if np.all(group_index == -1):
```

Comment:
can you type this function (inputs and outputs).

also add a comment here on all null groups

Reply:
Done.